packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1002,6 +1002,119 @@ class _SliverAppBarDelegate extends SliverPersistentHeaderDelegate {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=freeform}
+///
+/// This sample shows a [SliverAppBar] with three simple properties. The first property
+/// is [pinned],The second property is [snap] while the third property is [floating].
+///
+/// ```dart imports
+/// import 'package:flutter/material.dart';
+/// ```
+///
+/// ```dart
+/// void main() => runApp(MyApp());
+///
+/// class MyApp extends StatefulWidget {
+///   const MyApp({Key key}) : super(key: key);
+///
+///   @override
+///   State<StatefulWidget> createState() => _MyAppState();
+/// }
+///
+/// class _MyAppState extends State<MyApp> {
+///   bool _pinned = true;
+///   bool _snap = false;
+///   bool _floating = false;
+///
+///   // SliverAppBar is declared in Scaffold.body, in slivers of a
+///   // CustomScrollView.
+///   @override
+///   Widget build(BuildContext context) {
+///     return MaterialApp(
+///       home: Scaffold(
+///         body: CustomScrollView(
+///           slivers: <Widget>[
+///             SliverAppBar(
+///               pinned: this._pinned,
+///               snap: this._snap,
+///               floating: this._floating,
+///               expandedHeight: 160.0,
+///               flexibleSpace: FlexibleSpaceBar(
+///                 title: const Text("SliverAppBar"),
+///                 background: FlutterLogo(),
+///               ),
+///             ),
+///             SliverToBoxAdapter(
+///               child: Center(
+///                 child: Container(
+///                   height: 2000,
+///                   child: Text("Scroll to see SliverAppBar in effect ."),
+///                 ),
+///               ),
+///             ),
+///           ],
+///         ),
+///         bottomNavigationBar: BottomAppBar(
+///           child: ButtonBar(
+///             alignment: MainAxisAlignment.spaceEvenly,
+///             children: <Widget>[
+///               Row(
+///                 children: <Widget>[
+///                   Text('pinned'),
```

Comment:
Hello,
can you please add the `const` keyword to the `Text` widget?
Thanks

Reply:
Ok sure